### PR TITLE
Fix Ide file and terminal icons

### PR DESCRIPTION
**Pull request by hakankaradis**
Fix styling in ide for editor and terminal icons.
Renaming file from editor fix
Add blue tag to active tab in all splits.

### CODE REVIEW

**Comment by ftasdelen** [comment]
@hakankaradis Can we make the second pane active tab icon faded instead of white?

image

Can you also make the first pane selected tab file name white?

image

File: client/ide/lib/index.coffee [hunk 1]
```diff
@@ -258,6 +258,14 @@ class IDEAppController extends AppController
 
   setActiveTabView: (tabView) ->
 
+    activePane = tabView.getActivePane()
+
+    for view in @ideViews
+      for tab in view.holderView.tabs.subViews
+        if tab is activePane?.tabHandle
+        then activePane.tabHandle.setClass 'focus'
+        else tab.unsetClass 'focus'
```
Comment (gokmen) [inline client/ide/lib/index.coffee]: we shouldn't need something like that, currently there is a style change on focus in somewhere else. I might be wrong but this looks weird to me.

Comment (gokmen) [inline client/ide/lib/index.coffee]: cc/ @sinan

Comment (sinan) [inline client/ide/lib/index.coffee]: yeah, which is `active` class.

https://github.com/koding/kd/blob/master/lib/components/tabs/tabview.coffee#L240

https://github.com/koding/kd/blob/master/lib/components/tabs/tabhandleview.coffee#L58

**Comment by sinan** [comment]
@hakankaradis can you post a little gif showing you switch tabs please?

File: client/ide/lib/index.coffee (hunk 1 above)
Comment (hakankaradis) [inline client/ide/lib/index.coffee]: We will need one global class name across all panes across all split views.

**Comment by ftasdelen** [comment]
Looks good to me.

File: client/ide/lib/index.coffee (hunk 1 above)
Comment (gokmen) [inline client/ide/lib/index.coffee]: merging this one but still looks suspicious to me :)

**Review by gokmen — APPROVED** [review]

**Pull request merged**